docs/releases/1.10.1.txt

```diff
@@ -61,3 +61,7 @@ Bugfixes
 
 * Fixed ``contrib.admindocs`` crash when a view is in a class, such as some of
   the admin views (:ticket:`27018`).
+
+* Fixed the ``migrate`` command to include already applied migrations state
```

Comment:
It could be useful to translate this migrations terminology into something less technical regarding the bugs that this fixes.